roles/config-rh-sso/tasks/setup-rh-sso.yml

```diff
@@ -24,4 +61,4 @@
 
 - name: 'Waiting for SSO service to load'
```

Comment:
Do we have an open issue to track this to make it "smarter" to avoid just doing a sleep?

Reply:
No. I will create one for this.

Reply:
Issue #480 has been created for this.